apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -362,6 +401,18 @@ loginView self@{ state: state@{ logger }, setState } = React.fragment
               Right user -> do
                 setState $ setLoggedInUser $ Just user
                 logger.setUser $ Just user
+                path <- pathname =<< location =<< window
+                case path of
+                  "/fakturor" -> Aff.launchAff_ $ withSpinner (setState <<< setLoading) do
```

Comment:
I don't really like pattern matching on the location when react router kind of does that for us already. Of course the problem here is where to fetch the payments.

How do you feel about passing `Aff (Array SubscriptionPayments)` to `PaymentAccordion` props and fetching them in the `didMount` function there? 🤔

Reply:
I moved it to `paymentsView` and used `didMount` and it's much nicer this way.